.azure-pipelines.yml

```diff
@@ -203,4 +203,4 @@ stages:
       displayName: 'test'
       env:
         AZURE_ACCESS_TOKEN: "$(System.AccessToken)"
-        TFLAGS: "-vc /usr/bin/curl.exe -r -rm $(tests)"
+        TFLAGS: "-r -rm $(tests)"
```

Comment:
This change seems to be unrelated.

Reply:
This is why it is in a temporary commit: this commit allows me to enable functional CI tests for the PR. It is intended to be removed before pulling to master.

Reply:
Actually I think you are right about removing that parameter and if you would remove it while merging to master, the ManageSieve tests would be disabled again, because the verification of the test server would not work anymore.

Reply:
Yes, that's it! At least until /usr/bin/curl.exe supports ManageSieve.
That said, I don't think it impacts the test results, as an unsupported protocol seems not be counted as a failure. ManageSieve will not be tested, that's all.
In the current PR, I really need it to be tested: that's why I made the temp change.
I did not follow why this `--vc` option has been enabled, but I can imagine it will force performing tests even if the tested version is not able to verify server.
This option should probably be kept, as we do not implement a new protocol every day :-)

Reply:
I added that parameter some time ago, because the curl given there will also be used to communicate with the Azure Pipelines test results API and a broken build would cause the communication with the API to fail.

Reply:
I see. So let's keep it.
Paradoxically the real problem here is the tests came too soon after a feature as been implemented !

Reply:
`-vc /usr/bin/curl.exe` should only be done while debugging, we shouldn't have it the CI all the time because of exactly the reason @monnerat found out: the old version might not do the thing the current test script wants.

Reply:
@mback2k : as you mastered the Azure Pipeline and I'm not very skilled about it, I prefer to let you resolve this (outside this unrelated PR,  of course!).

Reply:
Will do, see https://github.com/curl/curl/pull/7591.